cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -117,7 +117,8 @@ func newHcsStandaloneTask(ctx context.Context, events publisher, req *task.Creat
 }
 
 // createContainer is a generic call to return either a process/hypervisor isolated container, or a job container
-//  based on what is set in the OCI spec.
+//
+//	based on what is set in the OCI spec.
```

Comment:
What is going on here?

Reply:
Weird new gofmt rule I believe..

Reply:
I'm on 1.19, I don't remember this prior. Believe it's due to their being two spaces in between // and the message